lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,29 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        if not qs:
+            default_onboarding = self.model()
+            default_onboarding.save()
+
+            qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        return qs
+
+    def has_add_permission(self, request):
+        return self.model.objects.all().count() == 0
+
+    def get_actions(self, request):
+        delete_action = 'delete_selected'
```

Comment:
Docstring. Completely not obvious what is happening here.

Reply:
Done.